--- origsrc/libedit-20170329-3.1/src/editline/readline.h	2017-03-29 13:15:04.000000000 -0500
+++ src/libedit-20170329-3.1/src/editline/readline.h	2017-11-06 13:17:46.008511200 -0600
@@ -74,7 +74,7 @@ typedef KEYMAP_ENTRY *Keymap;
 
 #ifndef CTRL
 #include <sys/ioctl.h>
-#if !defined(__sun) && !defined(__hpux) && !defined(_AIX)
+#if !defined(__sun) && !defined(__hpux) && !defined(_AIX) && !defined(__CYGWIN__)
 #include <sys/ttydefaults.h>
 #endif
 #ifndef CTRL
